lib/libs/email/content/upload-subsequent-documents/emailTemplates/AppKCMS.tsx

```diff
@@ -0,0 +1,39 @@
+import { CommonEmailVariables, Events } from "shared-types";
+import {
+  PackageDetails,
+  BasicFooter,
+  Attachments,
+  SubDocHowToAccess,
+  Divider,
+} from "../../email-components";
+import { BaseEmailTemplate } from "../../email-templates";
+import { styles } from "../../email-styles";
+import { Text } from "@react-email/components";
+
+export const AppKCMSEmail = (props: {
+  variables: Events["UploadSubsequentDocuments"] & CommonEmailVariables;
+}) => {
+  const variables = props.variables;
+  const previewText = `Action required: review new documents for 1915(c) ${variables.id} in OneMAC.`;
+  const heading = `New documents have been submitted for 1915(c) ${variables.id} in OneMAC.`;
+  return (
+    <BaseEmailTemplate
+      previewText={previewText}
+      heading={heading}
+      applicationEndpointUrl={variables.applicationEndpointUrl}
+      footerContent={<BasicFooter />}
+    >
+      <PackageDetails
+        details={{
+          "State or territory": variables.territory,
```

Comment:
```suggestion
          "State or Territory": variables.territory,
```

Dunno if you were by sprint review, but they've asked territory to be capitalized to avoid offending our territories

Reply:
I didn't see that, but thanks for letting me know! I made this change and I also see you have a PR open for this very thing so I will take a look 👀